nats-subscriber/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:8
+WORKDIR /app
+COPY package.json /app
+RUN npm install
+COPY . /app
+
+ENV LFH_NATS_SUBSCRIBER_HOST=nats-server
```

Comment:
We will want to document all of the ENV variables in the "dockerfile header comments" as we're doing with other images.